ethsigner/signingapi/build.gradle

```diff
@@ -22,3 +22,9 @@ jar {
       )
   }
 }
+
+dependencies {
+  implementation 'org.apache.logging.log4j:log4j-api'
+  implementation 'org.apache.logging.log4j:log4j-core'
+  implementation 'org.apache.logging.log4j:log4j-slf4j-impl'
```

Comment:
Thinking about it, the `core` and `slff4j-impl` should be `runtime` dependencies, as there should be no compile time dependencies (Pantheon has these as `runtime` dependencies`)

Reply:
Done, changed all build.gradle files